package.json

```
@@ -38,8 +38,9 @@
     "pack:dir": "npm run clean:dist && electron-builder --dir -p never",
     "pack:lin": "npm run clean:dist && electron-builder --linux --x64 -p never",
     "pack:mac": "npm run clean:dist && electron-builder --mac -p never",
-    "pack:mac:mas": "npm run clean:dist && electron-builder --mac mas -p never",
-    "pack:mac:masdev": "npm run clean:dist && electron-builder --mac mas-dev -p never",
+    "pack:mac_arm64": "npm run clean:dist && electron-builder --mac --arm64 -p never",
```

Nitpick but for the other jobs we use `:`, not `_`. @joseph-flinn Something you could fix while hooking up the CI jobs perhaps?

---

I would prefer we keep consistency here across the board. @nitrocode , can you please fix this syntax?
```suggestion
    "pack:mac:arm64": "npm run clean:dist && electron-builder --mac --arm64 -p never",
```